testsuite/summarize.awk

```diff
@@ -138,14 +138,9 @@ function record_unexp() {
     record_unexp();
 }
 
-/^re-ran / {
-    if (in_test){
-        printf("error at line %d: found re-ran inside a test\n", NR);
-        errored = 1;
-    }else{
-        RERAN[substr($0, 8, length($0)-7)] += 1;
-        ++ reran;
-    }
```

Comment:
Below there is this block:
```awk
        if (reran != 0){
            printf("  %3d test dir re-runs\n", reran);
        }
```
which also wants deleting. It turns out there are some other vestiges of that old system present in `testsuite/Makefile` as well in the `default` target, where these four lines can all be deleted:

```Makefile
        @echo
        @echo "all*, parallel* and list can automatically re-run failed test"
        @echo "directories if MAX_TESTSUITE_DIR_RETRIES permits"
        @echo "(default value = $(MAX_TESTSUITE_DIR_RETRIES))"
```